lib/ddtrace/profiling/scheduler.rb

```diff
@@ -0,0 +1,90 @@
+require 'ddtrace/worker'
+require 'ddtrace/workers/polling'
+
+module Datadog
+  module Profiling
+    # Pulls and exports profiling data on an async interval basis
+    class Scheduler < Worker
+      include Workers::Polling
+
+      DEFAULT_INTERVAL = 60
+      MIN_INTERVAL = 15
```

Comment:
That should be 0 as we want profiles to be as close to 1 minute as possible.

Reply:
Set to 0.